public/js/saveApplicant.js

```diff
@@ -32,26 +71,32 @@ let makeApiRequest = async (data) => {
         },
         method: 'post',
         body: JSON.stringify(data)
-    })
-        .then(response => response.json())
-        .then((data) => {
-            if (data.success) {
-                window.location.href = './admin';
-            } else {
-                document.getElementById('errorMsg').innerHTML = "Please contact administrator. Or <a href='./admin'>click here</a> to return to admin page"
-            }
-        })
-}
+    }).then(response => {
+        let generalErrorMessage = document.querySelector('#generalError');
+
+        if (response.status === 200) {
+            window.location.href = './admin';
+        } else if (response.status === 400) {
+            generalErrorMessage.innerHTML = "You must fill out all form options.";
+            generalErrorMessage.classList.remove('hidden');
+            generalErrorMessage.classList.add('alert-danger');
+        } else {
+            generalErrorMessage = "Something went wrong, please try again later.";
+            generalErrorMessage.classList.remove('hidden');
+            generalErrorMessage.classList.add('alert-danger');
+        }
```

Comment:
You're removing and adding the same class on both of these. You could just do it once after the `else` statement is closed.

Reply:
aha good shout